packages/backend/src/graphql/schema.graphql

```diff
@@ -698,10 +695,9 @@ type WalletAddress implements Model {
     "Ascending or descending order of creation."
     sortOrder: SortOrder
   ): WalletAddressKeyConnection
-}
 
-type AdditionalPropertyConnection {
-  properties: [AdditionalProperty!]!
+  "List additional properties associated with this wallet address."
+  additionalProperties: [AdditionalProperty]!
```

Comment:
No additional properties layer (since we aren't gonig to be doing any paging anyways), so its just:


```gql
additionalProperties {
  key
  value
visibleInOpenPayments
}
```